src/main.ts

```diff
@@ -21,25 +24,19 @@ declare module "hono" {
     interface ContextVariableMap extends HonoVariables {}
 }
 const app = new Hono();
-const konfigStore = await konfigLoader();
 
 let innertubeClient: Innertube;
 let innertubeClientFetchPlayer = true;
-const innertubeClientOauthEnabled = konfigStore.get(
-    "youtube_session.oauth_enabled",
-) as boolean;
-const innertubeClientJobPoTokenEnabled = konfigStore.get(
-    "jobs.youtube_session.po_token_enabled",
-) as boolean;
-const innertubeClientCookies = konfigStore.get(
-    "jobs.youtube_session.cookies",
```

Comment:
I believe this was incorrectly used as `jobs.youtube_session.cookies` and should have just been `youtube_session.cookies`. I've corrected that here.